README.md

```diff
@@ -136,6 +136,12 @@ This example just streams the body of the request. The body of the response can
 
 Check out the `examples` folder how your server could look like.
 
+The first parameter (often written as `RequestInterface $request` in examples and tests) 
+of the callback function and a middleware is a `ServerRequest` object based on the 
```

Comment:
I agree this text would need improvemtn but you can use it in the callback/middleware function.
Do you suggest to avoid the documentation of this in the README?